equinix/services/fabricv4/models/all_physical_ports_response.py

```diff
@@ -80,9 +80,9 @@ def to_dict(self) -> Dict[str, Any]:
         # override the default output from pydantic by calling `to_dict()` of each item in data (list)
         _items = []
         if self.data:
-            for _item in self.data:
-                if _item:
-                    _items.append(_item.to_dict())
+            for _item_data in self.data:
+                if _item_data:
+                    _items.append(_item_data.to_dict())
```

Comment:
These sorts of changes are a result of https://github.com/OpenAPITools/openapi-generator/pull/19223, which fixes a potential bug when models contain multiple list attributes with different element types.